components/automate-ui/src/styles.scss

```diff
@@ -204,3 +204,33 @@ chef-loading-spinner[fixed] {
 input {
   line-height: normal;
 }
+
+// chef-input directive styles here when NOT wrapped in chef-form-field
+.chef-input {
```

Comment:
@Shadae - at this point I almost feel like we should get rid of chef-input directive and just place this under input.  We want all of our inputs to be consistent across the site - so changing this to all be under <input> would make sense to me.